frontend/tests/controllers/TeamGroupsTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/**
+ * TeamGroupsTest
+ *
+ * @author juan.pablo
+ */
+
+class TeamGroupsTest extends \OmegaUp\Test\ControllerTestCase {
+    /**
+     * Basic create team group test
+     */
+    public function testCreateTeamGroup() {
+        ['identity' => $identity] = \OmegaUp\Test\Factories\User::createUser();
+        $name = \OmegaUp\Test\Utils::createRandomString();
+        $description = \OmegaUp\Test\Utils::createRandomString();
+        $alias = \OmegaUp\Test\Utils::createRandomString();
+
+        $login = self::login($identity);
+        $response = \OmegaUp\Controllers\Group::apiCreateTeamGroup(
+            new \OmegaUp\Request([
+                'auth_token' => $login->auth_token,
+                'name' => $name,
+                'alias' => $alias,
+                'description' => $description
+            ])
+        );
+
+        $this->assertEquals('ok', $response['status']);
+
+        $teamGroup = \OmegaUp\DAO\TeamGroups::getByName($name);
+        $this->assertNotNull($teamGroup);
+        $this->assertEquals($description, $teamGroup->description);
+        $this->assertTrue(
+            \OmegaUp\Authorization::isTeamGroupAdmin($identity, $teamGroup)
+        );
+
+        $teamGroup = \OmegaUp\DAO\TeamGroups::getByAlias($alias);
+        $this->assertNotNull($teamGroup);
+        $this->assertEquals($description, $teamGroup->description);
+    }
+
+    public function testCreateTeamGroupForNonAuthorizedUser() {
+        ['identity' => $identity] = \OmegaUp\Test\Factories\User::createUser();
+        [
+            'identity' => $nonAuthorizedIdentity,
+        ] = \OmegaUp\Test\Factories\User::createUser();
+        $name = \OmegaUp\Test\Utils::createRandomString();
+        $description = \OmegaUp\Test\Utils::createRandomString();
+        $alias = \OmegaUp\Test\Utils::createRandomString();
+
+        $login = self::login($identity);
+        $response = \OmegaUp\Controllers\Group::apiCreateTeamGroup(
```

Comment:
```suggestion
        \OmegaUp\Controllers\Group::apiCreateTeamGroup(
```

si un valor no se utiliza, no hay que "bautizarlo" con un nombre. igual en L90

Reply:
Listo